workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
```

Comment:
@cameel those values are just placeholders and have to be replaced.
@chriseth It will work without a working e-mail. But I would create a separate GitHub account for the bot, in the way how ReactJS did: https://github.com/reactjs-translation-bot. It looks much cooler :)